frontend/src/pages/articles/components/article-card/article-card.tsx

```diff
@@ -191,7 +192,10 @@ const ArticleCard: React.FC<Properties> = ({
           <h4 className={styles.title}>{title}</h4>
           <article
             className={getValidClassNames(styles.text, 'text-overflow')}
-            dangerouslySetInnerHTML={{ __html: sanitizeHtml(text) }}
+            style={{ fontSize: 14 }}
+            dangerouslySetInnerHTML={{
+              __html: sanitizeHtml(modifyFontSize(text, '14px')),
+            }}
```

Comment:
Can't we just make it with classes styling

Reply:
If with important we'd need just to provide styling (without replacing anything in text) - then I believe you can